Packs/Automox/Integrations/Automox/Automox.py

```diff
@@ -0,0 +1,851 @@
+            method="POST",
+            url_suffix=f"/servers/{device_id}/queues",
+            params=params,
+            data=payload,
+        )
+
+    def update_device(self, org_id, device_id, payload) -> List[Dict]:
+        params = {
+            "o": org_id,
+        }
+
+        return self._http_request(
+            method="PUT",
+            url_suffix=f"/servers/{device_id}",
+            params=params,
+            json_data=payload,
+            resp_type="response",
+        )
+
+    def update_group(self, org_id, group_id, payload) -> List[Dict]:
+        params = {
+            "o": org_id,
+        }
+
+        return self._http_request(
+            method="PUT",
+            url_suffix=f"/servergroups/{group_id}",
+            params=params,
+            json_data=payload,
+            resp_type="response",
+        )
+
+    def upload_vulnerability_sync_file(self, org_id, type, payload, files) -> List[Dict]:
+        return self._http_request(
+            method="POST",
+            url_suffix=f"/orgs/{org_id}/tasks/{type}/batches/upload",
+            data=payload,
+            files=files,
+        )
+
+    def get_group(self, org_id, group_id) -> List[Dict]:
+        params = {
+            "o": org_id,
+        }
+
+        return self._http_request(
+            method="GET",
+            url_suffix=f"/servergroups/{group_id}",
+            params=params
+        )
+
+    def get_device(self, org_id, device_id) -> List[Dict]:
+        params = {
+            "o": org_id,
+        }
+
+        return self._http_request(
+            method="GET",
+            url_suffix=f"/servers/{device_id}",
+            params=params
+        )
+
+    def create_group(self, org_id, payload) -> List[Dict]:
+        params = {
+            "o": org_id,
+        }
+
+        return self._http_request(
+            method="POST",
+            url_suffix=f"/servergroups",
+            params=params,
+            data=payload,
+        )
+
+''' HELPER FUNCTIONS '''
+
+def remove_keys(excluded_keys_list : List[str], data : List[Dict]) -> Dict[str, Any]:
+    for key_string in excluded_keys_list:
+        keys = key_string.split(".")
+        data = remove_key(keys, data)
+
+    return data
+
+def remove_key(keys_to_traverse, data):
+    try:
+        key = keys_to_traverse[0]
+
+        # If we've reached the last key in the list to traverse we can just drop it.
+        if len(keys_to_traverse) == 1:
+            del data[key]
+            return data
+
+        # Lists and dicts require us to move on with traversal.
+        if isinstance(data[key], dict):
+            data[key] = remove_key(keys_to_traverse[1:], data[key])
+        elif isinstance(data[key], list):
+            for i in range(len(data[key])):
+                data[key][i] = remove_key(keys_to_traverse[1:], data[key][i])
+        else:
+            del data[key]
+
+    except Exception as e:
+        demisto.error(f"Key '{key}' not found in Automox response.")
```

Comment:
If key isn't in response is it really an error for the user? Seems I could see debug for troubleshooting if needed but this could be noising in the cases the key isn't there and not actually be an issue for the end user. Not sure how these are exposed but just a though

Reply:
This actually just sends it to the logs with loglevel "error". It doesn't surface for the user.